freeipa/src/main/java/com/sequenceiq/freeipa/cost/FreeIpaCostService.java

```diff
@@ -44,7 +50,10 @@ public Map<String, RealTimeCost> getCosts(List<String> environmentCrns) {
         Map<String, RealTimeCost> realTimeCosts = new HashMap<>();
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
 
-        List<Stack> stacks = stackService.getMultipleDistinctByEnvironmentCrnsAndAccountIdWithList(environmentCrns, accountId);
+        List<Stack> stacks = stackService.getMultipleDistinctByEnvironmentCrnsAndAccountIdWithList(environmentCrns, accountId)
```

Comment:
same here, maybe worth moving the filtering to db side

Reply:
https://github.com/hortonworks/cloudbreak/pull/14060